### ISSUE TITLE
Computer Vision

### ISSUE THREAD

**Issue by WillNilges** [WillNilges/Kudos#2]
- Facial Rec
- Pathfinding
- Navigation

**Comment by hp77-creator** [comment]
Can you explain a bit? What do you mean by Facial Rec, I mean how is facial rec going to be implemented? I would like to contribute in solving this issue.

**Comment by WillNilges** [comment]
@hp77-creator
Hey there,
We're not really sure yet. this branch of the project is in very early stages, and we've only written a few dozen lines of code so far. I can tell you it'll be OpenCV on an Nvidia Jetson Nano. We'll probably be using eigenfaces for individual user recognition. 
https://www.mitpressjournals.org/doi/10.1162/jocn.1991.3.1.71

https://www.docs.opencv.org/2.4/modules/contrib/doc/facerec/facerec_tutorial.html#eigenfaces-in-opencv

The TL;DR of this project is to learn more about OpenCV and its applications for automated robotics. Part of that is allowing the recognition of individual users, which could be a neat gimmick at a makerfaire.

**Comment by hp77-creator** [comment]
@WillNilges 
Well, I asked because I am a kind of beginner in the facial recognition with OpenCV too. And I am looking for projects to get hands-on. If you need an extra hand, just post an issue. I would like to work on it.

**Comment by WillNilges** [comment]
Sure thing. Keep on the lookout.
